File: Number of Subarrays of 0's.py
class Solution():
    def no_of_subarrays(self, n, arr):
        count = 0
        answer = 0
        for i in range(n):
            if arr[i] == 0:
                count += 1
            else:
                answer += int((count*(count+1))/2)
                count = 0
        answer += int((count*(count+1))/2)
        return answer

File: Number of Subarrays of 0's.cpp
long long int no_of_subarrays(int n, vector<int> &arr) {

    long long int count = 0;
    long long int answer = 0;
    for (int i = 0; i < n; i++) {
        
        if(arr[i] == 0) {
            count++;
        }
        else {
            answer += (count*(count+1))/2;
            count= 0;
        }
    }
    answer += (count*(count+1))/2;
    return answer;
}
